### PR TITLE
723 cypress axe wcag testing

### PR DESCRIPTION
**Pull request by louiseClarkUSDS**

### CODE REVIEW

**Review by SteveMattinglyUSDS — DISMISSED** [review]
I don't see any problems here, but I have a few questions or maybe suggestions.
First, did you mean to PR to `staging` rather than `master`?
Second, I wonder why you created a separate file for each of your tests. It works, but putting them in one file would probably be more typical.
Finally, I came up with `cypress/e2e/sitemap.sh`, a script that generates a JSON file of all site pages by processing the sitemap. That way I can test all pages-- even future ones-- without having to code for each. Let me know if you'd like to talk about using that approach.

**Review by SteveMattinglyUSDS — APPROVED** [review]
👍

File: cypress/e2e/accessibility/07_accessibility.cy.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const urls = require('../sitemap.json'); // see sitemap.sh
+
+describe('Entire Website accessibility', () => {
+  urls.forEach((url) => {
+    it(`${url.length === 0 ? '/' : url}`, () => {
+      // Wait for page to load ...
+      cy.intercept(url).as('getPage');
+      cy.visit(url);
+      cy.checkPageA11y(url);
```
Comment (SteveMattinglyUSDS) [inline cypress/e2e/accessibility/07_accessibility.cy.js]: You want to use a `wait` with the @name of the intercept you are doing, like this:
```
      cy.intercept(url).as('getPage');
      cy.visit(url);
      cy.wait('@getPage').then(() => {
        cy.checkPageA11y(url);
      });
```
This ensures that the page is completely loaded before you run your checks.

**Review by SteveMattinglyUSDS — CHANGES_REQUESTED** [review]

File: cypress/e2e/accessibility/07_accessibility.cy.js (hunk 1 above)
Comment (louiseClarkUSDS) [reply cypress/e2e/accessibility/07_accessibility.cy.js]: Perfect! I'll make that change.

**Pull request merged**